nodl_python/nodl/schemas/v1.xsd

```diff
@@ -49,7 +51,7 @@
     <xs:element name="topic">
         <xs:complexType>
             <xs:sequence>
-                <xs:element ref="qos"/>
+                <xs:element name="qos_profile" type="qos_type"/>
```

Comment:
Do we need a `minOccurs` here as well?

Reply:
No, sequence defaults max and min to 1

Reply:
Ah right, you're just making it optional for the service, gotcha.